modulemd/v1/modulemd-validator.c

```diff
@@ -178,10 +281,7 @@ main (int argc, char *argv[])
               for (gsize i = 0; i < failures->len; i++)
                 {
                   doc = (ModulemdSubdocument *)g_ptr_array_index (failures, i);
-                  fprintf (stdout,
-                           "\nFailed subdocument (%s): \n%s\n",
-                           modulemd_subdocument_get_gerror (doc)->message,
-                           modulemd_subdocument_get_yaml (doc));
+                  summarize_failures (doc, fedmod);
```

Comment:
So, `fedmod lint` only runs on the set of YAML subdocuments that failed parsing? I'm not sure I see the value in adding it to this at all, then.

It seems to me that `fedmod lint` is more useful when run on its own against the complete YAML document, not just those that already failed the simple parser.